docs/data-collection/tear-down.md

```diff
@@ -2,7 +2,7 @@
 ## Showing the participant out
 
 - [ ] Extract the bed from the scanner's bore using the scanner's control wheel.
-    The home button can be used to bring the bed out.
+    The home (:fontawesome-solid-house:) button can alternatively be used to bring the bed out.
```

Comment:
Several things are repated or suggested later, but should be here:

After line 22
```
- [ ] Prepare the belt on the bed to be removed from the room when you show the participant out.
```

After line 23 (after adding the above line, 24)
```
- [ ] Disconnect the last section of the cannula and dispose of it in the trash can.
```